src/sections/Navbar.html

```diff
@@ -0,0 +1,22 @@
+<nav
+   class="flex flex-row justify-between items-center md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+>
+   <div class="flex flex-col text-[1rem] md:text-2xl lg:text-2xl">
```

Comment:
You can remove the 1rem arbitrary value; It doesn't change anything since the text is already 16px by default.